identity/token.go

```diff
@@ -19,15 +19,17 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/mendersoftware/go-lib-micro/addons"
 	"github.com/pkg/errors"
 )
 
 type Identity struct {
-	Subject  string `json:"sub" valid:"required"`
-	Tenant   string `json:"mender.tenant,omitempty"`
-	IsUser   bool   `json:"mender.user,omitempty"`
-	IsDevice bool   `json:"mender.device,omitempty"`
-	Plan     string `json:"mender.plan,omitempty"`
+	Subject  string         `json:"sub" valid:"required"`
+	Tenant   string         `json:"mender.tenant,omitempty"`
+	IsUser   bool           `json:"mender.user,omitempty"`
+	IsDevice bool           `json:"mender.device,omitempty"`
+	Plan     string         `json:"mender.plan,omitempty"`
+	Addons   []addons.Addon `json:"mender.addons,omitempty"`
```

Comment:
(e.g. by creating a `json.Marshaller` & `json.Unmarshaller` interface for `type Addons []Addon`)

Reply:
do you mind if I keep it that way?
there's no strong technical reason, but e.g. in tenantadm I commited to have every available addon clearly spelled out (eanbled or disabled).
it might be easier to think 'all addons are always there' on every part of the system...